src/Scientist/Internals/Experiment.cs

```diff
@@ -68,20 +69,19 @@ public ExperimentInstance(string name, Func<Task<T>> control, Func<Task<T>> cand
 
         static async Task<ExperimentResult> Run(Func<Task<T>> experimentCase)
         {
-            var sw = new Stopwatch();
-            sw.Start();
+            Chrono chrono = new Chrono();
```

Comment:
Hmmm good question. I had been messing with passing in an object by ref (which would have needed instance) it may be a leftover from that. Will check.